packages/nodes-base/nodes/MySql/v2/helpers/utils.ts

```diff
@@ -261,7 +274,13 @@ export function configureQueryRunner(
 					try {
 						const { query, values } = queryWithValues;
 						formatedQuery = connection.format(query, values);
-						const statements = formatedQuery.split(';').map((q) => q.trim());
+
+						let statements;
+						if ((options?.nodeVersion as number) <= 2.3) {
+							statements = formatedQuery.split(';').map((q) => q.trim());
+						} else {
+							statements = splitQueryToStatements(formatedQuery, false);
```

Comment:
maybe we could pass `nodeVersion` to `splitQueryToStatements`, so we don't have to repeat the lines above

Reply:
I would prefer not to do that as there there are some differences between batch modes that should not be a part of `splitQueryToStatements` helper